From 7afb79a21a9245b47b01886e56385fd2a19d6f50 Mon Sep 17 00:00:00 2001
From: Basheer Ahmed Muddebihal <basheer.ahmed.muddebihal@linux.intel.com>
Date: Wed, 13 Dec 2023 12:34:23 -0800
Subject: [PATCH] fpga: dfl: Fix the dfl dev type to Privare Feature

The device type was set to Port Type and changed to Private to reflect the correct
device type.

Signed-off-by: Basheer Ahmed Muddebihal <basheer.ahmed.muddebihal@linux.intel.com>
---
 drivers/fpga/dfl.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/fpga/dfl.c b/drivers/fpga/dfl.c
index 5ce4c6c16ede..97cbac8e1b30 100644
--- a/drivers/fpga/dfl.c
+++ b/drivers/fpga/dfl.c
@@ -68,7 +68,7 @@ static struct dfl_dev_info dfl_devs[] = {
 	{.name = DFL_FPGA_FEATURE_DEV_PORT, .dfh_id = DFH_ID_FIU_PORT,
 	 .devt_type = DFL_FPGA_DEVT_PORT},
 	{.name = DFL_FPGA_FEATURE_DEV_PRIV_FEAT, .dfh_id = DFH_ID_FIU_PRIV_FEAT,
-	 .devt_type = DFL_FPGA_DEVT_PORT},
+	 .devt_type = DFL_FPGA_DEVT_PRIV_FEAT},
 };
 
 /**
